notebooks/search/05-query-rules.ipynb

```diff
@@ -0,0 +1,437 @@
+      "metadata": {
+        "id": "0241694c"
+      },
+      "source": [
+        "## Initialize the Elasticsearch client\n",
+        "\n",
+        "Now we can instantiate the [Elasticsearch python client](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/index.html), providing the cloud id and password in your deployment."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 30,
+      "id": "f38e0397",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "f38e0397",
+        "outputId": "ad6df489-d242-4229-a42a-39c5ca19d124"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Elastic Cloud ID··········\n",
+            "Elastic Password··········\n"
+          ]
+        }
+      ],
+      "source": [
+        "from elasticsearch import Elasticsearch\n",
+        "from getpass import getpass\n",
+        "\n",
+        "CLOUD_ID = getpass(\"Elastic Cloud ID\")\n",
+        "CLOUD_PASSWORD = getpass(\"Elastic Password\")\n",
+        "\n",
+        "# Create the client instance\n",
+        "client = Elasticsearch(\n",
+        "    cloud_id=CLOUD_ID,\n",
+        "    basic_auth=(\"elastic\", CLOUD_PASSWORD)\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "fcd165fa",
+      "metadata": {
+        "id": "fcd165fa"
+      },
+      "source": [
+        "If you're running Elasticsearch locally or self-managed, you can pass in the Elasticsearch host instead. [Read more](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/connecting.html#_verifying_https_with_certificate_fingerprints_python_3_10_or_later) on how to connect to Elasticsearch locally."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "1462ebd8",
+      "metadata": {
+        "id": "1462ebd8"
+      },
+      "source": [
+        "Confirm that the client has connected with this test."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 31,
+      "id": "25c618eb",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "25c618eb",
+        "outputId": "30a6ba5b-5109-4457-ddfe-5633a077ca9b"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "{'name': 'instance-0000000000', 'cluster_name': '1a56ad21587c44d3930932eb9fa1d8e8', 'cluster_uuid': 'gX4zlwtlR4qhZpp1SPm4Yg', 'version': {'number': '8.8.2', 'build_flavor': 'default', 'build_type': 'docker', 'build_hash': '98e1271edf932a480e4262a471281f1ee295ce6b', 'build_date': '2023-06-26T05:16:16.196344851Z', 'build_snapshot': False, 'lucene_version': '9.6.0', 'minimum_wire_compatibility_version': '7.17.0', 'minimum_index_compatibility_version': '7.0.0'}, 'tagline': 'You Know, for Search'}\n"
```

Comment:
This suggests that Elasticsearch 8.8 is enough to use query rules. Should we say somewhere that this was introduced in Elasticsearch 8.10?